src/main/kotlin/com/connexin/ice/wrapper/service/AbstractEngine.kt

```diff
@@ -40,7 +40,7 @@ abstract class AbstractEngine() {
                 )
             }
         }
-        vaccineReport.indicators.forEach {
+        vaccineReport.indicators.filter { supportedIceWrapperInterpretations.contains(it.interpretation) }.forEach {
             obs.add(
                 VMRObservationResult(
                     it.code, it.system,
```

Comment:
looks like the  else -> ObservationConcept.PROOF_OF_IMMUNITY will never get hit after the filter